### PR TITLE
Fixed cv.plot()

### PR DESCRIPTION
**Pull request by DunklesArchipel**
Added a short docu on `def x_label()`

### CODE REVIEW

File: svgdigitizer/electrochemistry/cv.py [hunk 1]
```diff
@@ -183,7 +187,7 @@ def _add_time_axis(self, df):
     def plot(self):
```
Comment (saraedum) [inline svgdigitizer/electrochemistry/cv.py:187]: Do we want to run plot as a doctest here to see that it works now?

File: svgdigitizer/electrochemistry/cv.py [hunk 2]
```diff
@@ -73,6 +73,10 @@ def get_axis_unit(cls, unit):
 
     @property
     def x_label(self):
+        r"""
+        In contrast to the y-label, which only consists of a unit, the x-label consitst of a unit and a reference.
```
Comment (saraedum) [inline svgdigitizer/electrochemistry/cv.py:77]:
```suggestion
        Return the label on the x-axis of the SVG plot.
        
        In contrast to the y-label, which only consists of a unit, the x-label consists of a unit and a reference.
```

File: svgdigitizer/electrochemistry/cv.py (hunk 1 above)
Comment (DunklesArchipel) [reply svgdigitizer/electrochemistry/cv.py:187]: see #71

File: svgdigitizer/electrochemistry/cv.py (hunk 2 above)
Comment (saraedum) [reply svgdigitizer/electrochemistry/cv.py:77]: (there was a typo.)

**Review by saraedum — APPROVED** [review]

File: svgdigitizer/electrochemistry/cv.py [hunk 3]
```diff
@@ -73,6 +73,12 @@ def get_axis_unit(cls, unit):
 
     @property
     def x_label(self):
+        r"""
+        Return the label on the x-axis of the SVG plot.
+
```
Comment (DunklesArchipel) [inline svgdigitizer/electrochemistry/cv.py:78]:
```suggestion

```

Comment (saraedum) [reply svgdigitizer/electrochemistry/cv.py]: (my fault, sorry.)

**Pull request merged**